setup.py

```diff
@@ -221,27 +221,43 @@ def run(self):
         ''' run command '''
 
         has_errors = False
+        playbooks = set()
+        included_playbooks = set()
 
         for yaml_file in find_files(
                 os.path.join(os.getcwd(), 'playbooks', 'byo'),
                 None, None, r'\.ya?ml$'):
             with open(yaml_file, 'r') as contents:
-                for line in contents:
-                    # initialize_groups.yml is used to identify entry point playbooks
-                    if re.search(r'initialize_groups\.yml', line):
-                        print('-' * 60)
-                        print('Syntax checking playbook: %s' % yaml_file)
-                        try:
-                            subprocess.check_output(
-                                ['ansible-playbook', '-i localhost,',
-                                 '--syntax-check', yaml_file]
-                            )
-                        except subprocess.CalledProcessError as cpe:
-                            print('{}Execution failed: {}{}'.format(
-                                self.FAIL, cpe, self.ENDC))
-                            has_errors = True
-                        # Break for loop, no need to continue looping lines
-                        break
+                for task in yaml.safe_load(contents.read()):
+                    if not isinstance(task, dict):
+                        # Skip yaml files which do not contain plays or includes
+                        continue
+                    if 'include' in task.keys():
+                        # Add the playbook and capture included playbooks
+                        playbooks.add(yaml_file)
+                        included_file_name = task['include'].split()[0]
+                        included_file = os.path.normpath(
+                            os.path.join(os.path.dirname(yaml_file),
+                                         included_file_name))
+                        included_playbooks.add(included_file)
+                    elif 'hosts' in task.keys():
```

Comment:
Actually, `keys` is needed.  I ran into a case when the word `hosts` was merely present in the text of the `dict` as a whole.  This generated a false positive, so I was more specific in looking at the `keys` of the `dict` and not the whole `dict`.